PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+/// The Bar class
+public class Bar {
+    var yValues: [Double]
+    var label: String
+    var color: [UIColor]?
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    /// - parameter color: provide a color base on the class UIColor
+    public init(_ yValues: [Double], label: String, color: [UIColor]) {
+        self.yValues = yValues
+        self.label = label
+        self.color = color
+    }
+
+    /// - parameter yValues: Double array of y values
+    /// - parameter label: bar label
+    public init(_ yValues: [Double], label: String) {
+        self.yValues = yValues
+        self.label = label
+    }
+}
+
+/// PredixBarChartView -- Bar Chart
+open class PredixBarChartView: BarChartView {
+    /// :nodoc:
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        initialize()
+    }
+
+    /// :nodoc:
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        initialize()
+    }
+
+    /// Predix Mobile Bar chart initial values
+    fileprivate func initialize() {
+        legend.horizontalAlignment = .right
+        legend.verticalAlignment = .bottom
+        xAxis.labelPosition = .bottom
+        chartDescription?.enabled = false
+    }
+
+    /// Set message if there's no data provided
+    public func setNoDataText(message: String) {
+        noDataText = message
+    }
+
+    /// Add a limit line
+    /// - parameter limit: Double number of where the line should be drawn verticaly
+    /// - parameter label: String label of the line
+    public func addALimit(limit: Double, label: String) {
+        let ll = ChartLimitLine(limit: limit, label: label)
+        rightAxis.addLimitLine(ll)
+    }
+
+    /// Set the x axis label text color
+    /// - parameter uiColor: provide a color base on the class UIColor
+    public func setXAxisLabelTextColor(uiColor: UIColor) {
+        xAxis.labelTextColor = uiColor
+    }
+
+    /// Helper function to populate the Bar Chart based on each Bar data entry
```

Comment:
I wouldn't describe the main method in the class as a "Helper function"

More description, and an example of usage would be good here.